nvflare/tool/job/job_client_const.py

```diff
@@ -43,8 +43,12 @@
 CONFIG_CONF = "config.conf"
 
 DEFAULT_APP_NAME = "app"
+# for consistency, meta config have a dummy app name
+META_APP_NAME = "__meta__app__"
 APP_CONFIG_DIR = "config"
 
 APP_SCRIPT_KEY = "app_script"
 APP_CONFIG_KEY = "app_config"
 TEMPLATES_KEY = "templates"
+
+FILE_POSTFIX = [".py", ".json", ".jsonl", ".yml", ".yaml", ".conf"]
```

Comment:
Should we restrict this? What if sd contains checkpoint or binary files?

Reply:
@holgerroth do you want to copy over checkpoint or binary files?
Even if we copy all the thing here in job creation, in job submission we still have some constraint on what files can be submitted right?
@IsaacYangSLA @yhwen Do you remember what constraint we have?

Reply:
remove the restriction here in job creation, up to fl client/server side to accept / reject jobs